src/sentry/integrations/github/webhook.py

```diff
@@ -71,11 +71,15 @@
 logger = logging.getLogger("sentry.webhooks")
 
 
+# Functions that process webhook events need to have this signature.
+# This is used to type check the webhook processors.
 class WebhookProcessor(Protocol):
     def __call__(
         self,
         *,
-        event_type: str,
```

Comment:
All the functions implementing this protocol will be updated to this new signature.